xarray/core/groupby.py

```diff
@@ -98,14 +98,28 @@ def __init__(self, obj, group, squeeze=False, grouper=None):
             if the dimension is squeezed out.
         grouper : pd.Grouper, optional
             Used for grouping values along the `group` array.
+        group_bins : array-like, optional
```

Comment:
I'd prefer to standardize on calling this kwarg `bins` in all cases (e.g. `Groupby`, `da.groupby`, and  `pd.cut`).